### ISSUE TITLE
Frida-trace crash the 64bit debuggee 10.1.4 on windows 10

### ISSUE THREAD

**Issue by beyonddoor** [frida/frida-core#140]
Executes 64 bit notepad.exe, then to attach it:
`frida-trace -p 18760 -i MessageBoxW`
Notepad.exe 's process crashes, and frida-trace prints `Failed to start tracing: script is destroyed`.
But it works well on 32bit notepad.exe.
I use 64bit frida-server and test again, the result is the same, any helps?

**Comment by riverar** [comment]
Can't reproduce with 10.3.1, can you try upgrading? Also, what build/version of Windows 10?

**Comment by newlog** [comment]
The same happens to me with any binary in a Windows 7 box.
```

$ "C:\Users\user\AppData\Roaming\Python\Scripts\frida-trace.exe" -T -f "C:\Windows\System32\notepad.exe"

Instrumenting functions...
MoveWindow: Loaded handler at "C:\Users\user\Desktop\\__handlers__\MoveWindow.js"
GetFileInformationByHandle: Loaded handler at "C:\Users\Kevin Johnson\Desktop\__handlers__\GetFileInformationByHandle.js"
(...)
Failed to start tracing: script is destroyed
```

And the a message box is shown stating the app has crashed.

**Comment by newlog** [comment]
Honestly, after multiple tries with multiple binaries, frida-trace has crashed the programs I checked both in Windows 7 32 bits and Windows 10 64 bits.

I tried frida-trace with notepad.exe in both Windows 7 and 10. And in Windows 10 I tried it with a custom compiled x86 and x64 binary.

This is the syntax I used:

> $ "C:\Users\user\AppData\Roaming\Python\Scripts\frida-trace.exe" -T -f "C:\Windows\System32\notepad.exe"

In Windows 10 some functions were successfully traced before the crash. In Windows 7 I never got to trace anything.

:(
